ui-v2/src/lib/types.ts

```diff
@@ -1,7 +0,0 @@
-export type JSONValue =
-	| string
-	| number
-	| boolean
-	| Record<string, never>
-	| unknown[]
-	| null;
```

Comment:
This was moved to `ui-v2/src/components/variables/variable-dialog/variable-dialog.tsx` since it was only used in that location.